snaps/features/custom-ui.md

```diff
@@ -85,6 +116,37 @@ const content = panel([
 ]);
 ```
 
+### `form`
+
+Outputs a form for use in [interactive UI](./interactive-ui.md).
+
+#### Parameters
+
+An object with:
+
+- `name`: A string that will be sent to [`onUserInput`](../reference/entry-points.md#onuserinput) when the form is interacted with.
+- `children`: An array of [`input`](#input) or [`button`](#button) components.
+
+#### Example
+
+```js
+import { input, button, form } from '@metamask/snaps-sdk';
+
+const form = form({
+  name: 'form-to-fill',
+  children: [
+    input({
+      name: 'user-name',
+      placeholder: 'Your name',
+    }),
+    button({
+      value: 'Submit!',
+      type: 'submit'
```

Comment:
Is the parameter `type` or `buttonType`?

Reply:
Should be `buttonType`